src/Operations.jl

```diff
@@ -1023,7 +1023,7 @@ function rm(ctx::Context, pkgs::Vector{PackageSpec})
     prune_manifest(ctx.env)
     # update project & manifest
     write_env(ctx.env)
-    show_update(ctx)
+    show_update(ctx, ignore_indent=false)
```

Comment:
What is actually ignoring indent? It seems like these should not be necessary since false is the default?

Reply:
True. Removed. Thanks